packages/sak-app/src/index.tsx

```diff
@@ -11,15 +11,11 @@ import errorHandler from '@fpsak-frontend/error-api-redux';
 import AppIndex from './app/AppIndex';
 import configureStore from './configureStore';
 
-/* global VERSION:true */
 /* eslint no-undef: "error" */
 // @ts-ignore
-const release = VERSION;
 const environment = window.location.hostname;
-const isDevelopment = process.env.NODE_ENV === 'development';
 init({
-  dsn: isDevelopment ? 'http://dev@localhost:9000/1' : 'https://251afca29aa44d738b73f1ff5d78c67f@sentry.gc.nav.no/31',
-  release,
+  dsn: 'https://251afca29aa44d738b73f1ff5d78c67f@sentry.gc.nav.no/31',
```

Comment:
Betyr dette at vi begynner å gjøre kall mot sentry lokalt i utvikling også? Ønsker vi egentlig å gjøre det (uavhengig av om sentry filtrerer det bort)? 🤔 Feil som skjer i utvikling vil jo ikke være av interesse for andre enn den som sitter i det utviklingsmiljøet.

Reply:
Enig. fikset